pkg/api/defaults.go

```diff
@@ -460,22 +460,17 @@ func (p *Properties) setAgentProfileDefaults(isUpgrade, isScale bool) {
 					if profile.OSDiskSizeGB != 0 && profile.OSDiskSizeGB < VHDDiskSizeAKS {
 						profile.Distro = Ubuntu
 					} else {
-						if profile.IsNSeriesSKU() {
```

Comment:
validate.go and docs also need to be changed

Reply:
clusterdefinitions.md still needs to be updated (two places reference the docker-engine distro and the restrictions on GPU) as well as gpu.md

Reply:
Done